Inferences/Induction.cpp

```diff
@@ -23,21 +23,24 @@
 #include "Indexing/ResultSubstitution.hpp"
 #include "Lib/BitUtils.hpp"
 #include "Lib/DHMap.hpp"
+#include "Lib/DHSet.hpp"
```

Comment:
`DHSet.hpp` includes `DHMap.hpp`, so this can be a replacement instead of adding one more include.

Reply:
Done.